test/e2e_node/util.go

```diff
@@ -57,6 +58,9 @@ var startServices = flag.Bool("start-services", true, "If true, start local node
 var stopServices = flag.Bool("stop-services", true, "If true, stop local node services after running tests")
 var busyboxImage = "busybox"
 
+// Kubelet internal cgroup name for node allocatable cgroup.
+const defaultNodeAllocatableCgroup = "kubepods"
```

Comment:
yes, but we don't configure it for testing.  The tests in pod_container_manager_test.go make the same assumption.

Reply:
Can we get this from the text context?

Reply:
I dont think we can get this from the test context.  It is a kubelet default.  I am actually just moving the constant to this file, because it doesn't build for some reason where it was...